tools/api-markdown-documenter/api-report/api-markdown-documenter.api.md

```diff
@@ -332,10 +348,10 @@ export interface MarkdownDocument {
 // @public
 export interface MarkdownDocumenterConfiguration extends PolicyOptions, RenderingPolicies {
     apiModel: ApiModel;
+    readonly logger?: Logger;
```

Comment:
nit: consider naming it `log`.

Reply:
If that is the convention we use elsewhere, I'm happy to emulate. But I generally prefer to name objects using nouns, and methods/functions as verbs. I emulated the method naming scheme from release tools here, but my personal preference would be something like "logInfo", "logWarning", etc. So the call would look like "logger.logWarning" ("<noun>.<verb>").

Do we have any team-wide conventions around this sort of thing? Just curious.

Reply:
(I think I'm going to leave this as is for the current PR, but I'm definitely happy to come back to this later)